src/RadarChart.js

```diff
@@ -316,17 +248,23 @@ var radarChart = function () {
                                 .merge(verticesJoin)
                                 .attr("r", polygonPointSize)
                                 .attr("cx", function(d, i) {
-                                    // console.log(d.coordinates.x);
                                     return d.coordinates.x;
                                 })
                                 .attr("cy", function(d, i) {
-                                    // console.log(d.coordinates.y);
                                     return d.coordinates.y;
                                 })
                                 .attr("fill", color(gColor))
                                 .attr("fill-opacity", 1)
-                                .on(over, showTooltip)
-                                .on(out, verticesTooltipHide);
+                                .on(over, function(d) {
+                                    verticesTooltip.style("opacity", 0.9)
```

Comment:
so I change to this and it won't work, not sure if i miss write something?
verticesTooltip
                                        .html("<strong>Race</strong>: " + d.axis + "<br />" +
                                        "<strong>Percentile</strong>: " + d.value.toFixed(2) + "<br />")
                                        .style('visibility', 'visible')
                                        .style("left", (d3.event.pageX) + "px")
                                        .style("top", (d3.event.pageY) + "px");